.github/workflows/execution-plan-pr.yml

```diff
@@ -3,6 +3,9 @@ name: "Execution Plan on Pull Request"
   pull_request:
     branches:
       - main
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
huh, I thought that is already how it worked. I guess I was wrong.

Reply:
No it never cancelled runs for stale commits.